src/main/java/com/conveyal/r5/analyst/ExtractingGridStatisticComputer.java

```diff
@@ -0,0 +1,67 @@
+package com.conveyal.r5.analyst;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.model.S3Object;
+import com.google.common.io.LittleEndianDataInputStream;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.zip.GZIPInputStream;
+
+/**
+ * Extract the nth sample from a grid. Mostly useful when the grid contains a distribution rather than samples.
+ */
+public class ExtractingGridStatisticComputer {
```

Comment:
Disregard previous comment, based on the duplicated header code it made sense to make a super class.